java/jsf/kubernetes/Application/charts/sampleapp/values.yaml

```diff
@@ -2,10 +2,10 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 fullnameOverride: sampleapp
-replicaCount: 1
+replicaCount: 2
```

Comment:
why do we need to increase the replica count.

Reply:
Just making the count same for every sample app